src/detext/model/sp_emb_model.py

```diff
@@ -1,15 +1,17 @@
 import tensorflow as tf
 
 
-class SparseLinearModel:
-    """Linear model that performs computation on sparse features"""
+class SparseEmbModel:
+    """Embedding model that performs embedding lookup and summation on sparse features"""
 
     def __init__(self,
                  num_wide_sp: int,
                  wide_ftrs_sp_idx: tf.Tensor,
+                 sp_emb_size: int,
                  wide_ftrs_sp_val: tf.Tensor = None,
+                 padding_idx: int = 0,
                  initializer=tf.contrib.layers.xavier_initializer()):
-        """ Computes a linear score given wide feature indices and values
+        """ Computes a embedding given wide feature indices and values
```

Comment:
suggests adding sparse input features related instruction to the training data format section in `TRAINING.md`

Reply:
Added in TRAINING.md in latest update